tasks/libs/common/utils.py

```diff
@@ -720,14 +734,13 @@ def is_pr_context(branch, pr_id, test_name):
 
 
 @contextmanager
-def collapsed_section(section_name):
+def section(section_name, collapsed=False):
```

Comment:
```suggestion
def gitlab_section(section_name, collapsed=False):
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Since this will be / is used in a lot of places, and we're using multiple CI provider, I would name it `gitlab_section` to avoid confusion. WDYT ?